src/Reflection/ReflectionPropertyHookType.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Roave\BetterReflection\Reflection;
+
+use PropertyHookType as CoreReflectionPropertyHookType;
+
+enum ReflectionPropertyHookType: string
```

Comment:
Overall OK with this type, although some minimal docs would be required. A link to the upstream RFC would suffice, for example.